packages/antd/src/widgets/SelectWidget/index.tsx

```diff
@@ -65,6 +68,23 @@ export default function SelectWidget<
   const extraProps = {
     name: id,
   };
+
+  const selectOptions: DefaultOptionType[] | undefined = useMemo(() => {
+    if (Array.isArray(enumOptions)) {
+      if (!multiple && schema.default === undefined) {
+        enumOptions.unshift({ value: '', label: placeholder || '' });
+      }
```

Comment:
Maybe it would be better to do something like this so you don't push into an object you don't own?:
```
const options = [...enumOptions];
if (!multiple && schema.default === undefined) {
  options.unshift({ value: '', label: placeholder || '' });
}
return options.map(({ value: optionValue, label: optionLabel }, index) => ({
...
```

Reply:
Or you can do what `semantic-ui` does or change their implementation to match this one?

Reply:
Doing this (& wrapping in a `useMemo`) seems to take care of it. I needed to flip-flop the order of `map` and `unshift` compared to your example because we do not want the placeholder option value to be the enum option index.